src/pages/GroupAdmin/components/UpdateGroupModal.tsx

```diff
@@ -55,7 +65,7 @@ const UpdateGroupModal = ({ group }: UpdateGroupModalProps) => {
             })}
             type='email'
           />
-          <SubmitButton disabled={updateGroup.isLoading} formState={formState}>
+          <SubmitButton className={classes.margin} disabled={updateGroup.isLoading} formState={formState}>
```

Comment:
Istedenfor `className` så kan du heller bruker `sx`-prop'en siden du kun trenger en marginTop. Da kan du også fjerne useStyles
```suggestion
          <SubmitButton sx={{ mt: 2 }} disabled={updateGroup.isLoading} formState={formState}>
```

Reply:
Will do!